create.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Create</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Create</h1>
+    <main>
+      <form action="">
+        <label for="your-question">Your question</label>
+        <textarea id="your-question" cols="10" rows="5"></textarea>
+        <label for="your-answer">Your answer</label>
+        <textarea id="you-answer" cols="10" rows="5"></textarea>
```

Comment:
here is a little typo in the id (you -> your)

Reply:
yes thanks, I fixed this